src/providers/bttv/BttvEmotes.cpp

```diff
@@ -182,6 +182,12 @@ void BttvEmotes::loadChannel(std::weak_ptr<Channel> channel,
                     shared->addMessage(makeSystemMessage(
                         "This channel has no BetterTTV channel emotes."));
             }
+            else if (result.status() == 404)
+            {
+                // BTTV emotes not found
+                shared->addMessage(makeSystemMessage(
```

Comment:
404 is an "expected error" in case the channel does not have any emotes. Checking for the error like you do is great, but I don't think we need to tell the users that no emotes are found.